Pycloud Deployer/modules.py

```diff
@@ -0,0 +1,120 @@
+def create_publicip_parameters(location):
+    return {
+        'sku': {
+            'name': 'Standard'
+        },
+        'location': location,
+        'public_ip_allocation_method': 'Static'
+    }
+
+def create_nic_parameters(location, subscription_id, group_name, subnet_name, server_number=''):
+    if server_number == '':
+        publicip_name = 'dbserver_publicip'
+    else:
+        publicip_name = f'webserver{server_number}_publicip'
+
+    return {
+        'location': location,
+        'ip_configurations': [{
+            'name': 'MyIpConfig',
+            'subnet': {
+                'id': f"/subscriptions/{subscription_id}/resourceGroups/{group_name}/providers/Microsoft.Network/virtualNetworks/{group_name}_network/subnets/{subnet_name}"
+            },
+            'public_ip_address': {
+                'id': f"/subscriptions/{subscription_id}/resourceGroups/{group_name}/providers/Microsoft.Network/publicIPAddresses/{publicip_name}"
+            }
+        }]
+    }
+
+def create_webserver_parameters(location, subscription_id, group_name, interface_name):
+    return {
+        'location': location,
+        'hardware_profile': {
+            'vm_size': 'Standard_B1s'
+        },
+        'storage_profile': {
+            'image_reference': {
+                'sku': '2019-Datacenter',
+                'publisher': 'MicrosoftWindowsServer',
+                'version': 'latest',
+                'offer': 'WindowsServer'
+            },
+            'os_disk': {
+                'caching': 'ReadWrite',
+                'managed_disk': {
+                    'storage_account_type': 'Standard_LRS'
+                },
+                'create_option': 'FromImage'
+            }
+        },
+        'os_profile': {
+            'admin_username': 'testuser',
+            'computer_name': 'Azureuser',
+            'admin_password': 'Azureuser123',
```

Comment:
Hardcoding admin usernames and passwords is a potential risk, maybe use environment variables.